### ISSUE TITLE
game corrupted save while trying to go to new home

### ISSUE THREAD

**Issue by FatalError88** [RhenaudTheLukark/CreateYourFrisk#119]
so i went through each character one by one till i got to asriel.
i immediatley went to the dog and went through.
then i got an error saying "you should try to get these encounters the normal way hint: talk to the dog (wich i did) " then i restartet CYF (because it was the only option) and now its getting me an error that my save is corrupted / saved on a newer version(im using the latest anyway) so now my save file is dead

**Comment by RhenaudTheLukark** [comment]
I've been made aware of the problem, I'll keep you updated on it!

**Comment by octuaOSdev** [comment]
> I've been made aware of the problem, I'll keep you updated on it!

thanks

**Comment by RhenaudTheLukark** [comment]
Right, I did get an update since.

As it turns out, the error is caused by an animation in the third fight using a sprite with a name which isn't capitalized right.
While it works on Windows, it doesn't on Linux due to its case sensitiveness!

I suggest modifying the related encounter so you can run it from CYF's mod selector! You'll have the true culprit for what's going on by running the encounter like this!

You can remove some code in the encounter itself so it allows you running it from the mod selection screen

**Comment by octuaOSdev** [comment]
> Right, I did get an update since.
> 
> As it turns out, the error is caused by an animation in the third fight using a sprite with a name which isn't capitalized right. While it works on Windows, it doesn't on Linux due to its case sensitiveness!
> 
> I suggest modifying the related encounter so you can run it from CYF's mod selector! You'll have the true culprit for what's going on by running the encounter like this!
> 
> You can remove some code in the encounter itself so it allows you running it from the mod selection screen

is there a way i can repair my save file?

**Comment by RhenaudTheLukark** [comment]
Your save file is NOT corrupted, please read my last message!

**Comment by FatalError88** [comment]
> Right, I did get an update since.
> 
> As it turns out, the error is caused by an animation in the third fight using a sprite with a name which isn't capitalized right. While it works on Windows, it doesn't on Linux due to its case sensitiveness!
> 
> I suggest modifying the related encounter so you can run it from CYF's mod selector! You'll have the true culprit for what's going on by running the encounter like this!
> 
> You can remove some code in the encounter itself so it allows you running it from the mod selection screen

yeah sorry for saying this late but... i AM on windows and thats where i got the error

**Comment by RhenaudTheLukark** [comment]
That's strange... In that case, without a way to reproduce it consistently, I'm unsure as to what I can do on my end.
If you didn't update CYF between the various versions, and if you saved at all somewhere, there should be no issues.
What we'll do is that with just that info, it's hard for me to debug what's going on, as that issue didn't happen on my end when I tried it. If it happens once more, I'll look more into it, and I'll link the new issue to this one, sounds good?

**Comment by FatalError88** [comment]
sure, thanks!
